README.md

```diff
@@ -18,7 +18,7 @@ This will build (and cache) the library packages and applications.
 You can start the template next.js app:
 
 ```bash
-cd apps/next-productstarter && pnpm run start
+cd apps/next-product-starter && pnpm run dev
 ```
```

Comment:
Starting template with `pnpm run dev` intsead of `pnpm run start` will enable hot reloading.